hack/deploy-karmada.sh

```diff
@@ -139,7 +139,7 @@ util::create_signing_certkey "" "${CERT_DIR}" front-proxy-ca front-proxy-ca '"cl
 util::create_signing_certkey "" "${CERT_DIR}" etcd-ca etcd-ca '"client auth","server auth"'
 # signs a certificate
 util::create_certkey "" "${CERT_DIR}" "ca" karmada system:admin "system:masters" kubernetes.default.svc "*.etcd.karmada-system.svc.cluster.local" "*.karmada-system.svc.cluster.local" "*.karmada-system.svc" "localhost" "127.0.0.1" "${interpreter_webhook_example_service_external_ip_address}"
-util::create_certkey "" "${CERT_DIR}" "ca" apiserver karmada-apiserver "" "*.etcd.karmada-system.svc.cluster.local" "*.karmada-system.svc.cluster.local" "*.karmada-system.svc" "localhost" "127.0.0.1"
+util::create_certkey "" "${CERT_DIR}" "ca" apiserver karmada-apiserver "" "*.etcd.karmada-system.svc.cluster.local" "*.karmada-system.svc.cluster.local" "*.karmada-system.svc" "localhost" "127.0.0.1" $(util::get_apiserver_ip_from_kubeconfig "${HOST_CLUSTER_NAME}")
```

Comment:
There are two key point:

1. The `insecure-skip-tls-verify` of `karmada-apiserver context` created by script is `true`, so `karmadactl token create` don't use `--print-register-command`.
```
[root@master67 ~]# karmadactl token create --print-register-command --kubeconfig  /root/.kube/config --karmada-context karmada-apiserver
error: failed to get register command, err: no CA certificates found in kubeconfig
```

2. Because `karmadactl register` need to validate the `cert` of `karmada-apiserver`, so it should be signed a valid external IP. The one solution is getting ip from kubeconfig like `$(util::get_apiserver_ip_from_kubeconfig "${HOST_CLUSTER_NAME}")`, the another one is making `karmadactl register` support to use `insecurity` option.